src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx

```diff
@@ -96,13 +97,35 @@ function LineNumberTooltip({ editor, keyModifiers }: Props) {
     var rect = editor.codeMirror.getWrapperElement().getBoundingClientRect();
     var topVisibleLine = editor.codeMirror.lineAtHeight(rect.top, "window");
     var bottomVisibleLine = editor.codeMirror.lineAtHeight(rect.bottom, "window");
-    dispatch(fetchHitCounts(source!.id, editor.codeMirror.lineAtHeight()));
-    dispatch(fetchHitCounts(source!.id, topVisibleLine - 10));
-    dispatch(fetchHitCounts(source!.id, bottomVisibleLine + 10));
+
+    const viewport = editor.editor.getViewport();
+    const { from: topViewportLine, to: bottomViewportLine } = viewport;
+    const centerLine = ((bottomViewportLine - topViewportLine) / 2) | 0;
```

Comment:
Is this supposed to be `||` or did you mean `| 0`? 🙃

Reply:
Intentional - that's a "force this to be an integer value", in case we ended up with something like 37.5.